drivers/bluetooth/hci/h5.c

```diff
@@ -535,6 +536,14 @@ static void bt_uart_isr(const struct device *unused, void *user_data)
 				}
 			}
 
+			buf_tailroom = net_buf_tailroom(h5.rx_buf);
+			if (sizeof(byte) > buf_tailroom) {
```

Comment:
It's usually more natural (and common) to put the constant on the right side of the expression, so `buf_tailroom < 1`

Reply:
I guess you're right :) 

I've only heard the reverse when it comes to `==` or `!=` though, as to avoid accidentally assigning instead of comparing. 

I'll update it so that `buf_tailroom` will always be on the left side for these comparison for consistency as well.